include/dlaf/blas/tile_extensions.h

```diff
@@ -69,11 +70,11 @@ template <class T>
 void add(cublasHandle_t handle, T alpha, const matrix::Tile<const T, Device::GPU>& tile_b,
```

Comment:
```suggestion
void add(whip::stream_t stream, T alpha, const matrix::Tile<const T, Device::GPU>& tile_b,
```
and then the dispatch type of this kernel can change to `Plain` (on line 84).

Reply:
I didn't want to break anything in case there is something that calls it directly.

Reply:
Hmm, but wouldn't it be worth updating those direct calls, if there are any? From a quick search it looks like there aren't any direct calls either (there are a few `tile::add(Policy...)` calls, but those would not be affected).

Reply:
fixed in https://github.com/eth-cscs/DLA-Future/pull/796/commits/3527122bd6d09c14ccd555a8a9847815405297fa